sparv/modules/hunpos/hunpos.py

```diff
@@ -106,28 +106,23 @@ def hunpos_model(model: ModelOutput = ModelOutput("hunpos/suc3_suc-tags_default-
     def test_hunpos(model):
         stdin = TOK_SEP.join(["jag", "och", "du"]) + SENT_SEP
         stdout, _ = util.system.call_binary(binary, [model.path], stdin, encoding="UTF-8")
-        logger.debug(stdout)
-
-    tmp_model = Model("hunpos/hunpos-model.tmp")
-    reg_model = (
-        "https://github.com/spraakbanken/sparv-models/raw/master/hunpos/suc3_suc-tags_default-setting_utf8.model")
-    mac_model =  (
-        "https://github.com/spraakbanken/sparv-models/raw/master/hunpos/suc3_suc-tags_default-setting_utf8-mac.model")
-
-    if platform == "darwin":
-        logger.info("Downloading hunpos model for MacOS")
-        tmp_model.download(mac_model)
+        logger.debug("Output from 'hunpos-tag' with test input:\n%s", stdout)
+
+    # Run "hunpos-tag -h" to check what version was installed
+    stdout, _ = util.system.call_binary(binary, ["-h"], allow_error=True)
+    logger.debug("Output from 'hunpos-tag -h': %s", stdout)
+    # Search for keyword "verbose" in help message
+    matchobj = re.match("verbose", stdout.decode())
```

Comment:
`re.search(...)`, not `match`

Minor detail: I suggest searching for `--verbose`, to make it clear for any future reader of the code that it's a flag we're looking for

Reply:
✅